QUALITY_DECLARATION.md

```diff
@@ -1,132 +1,194 @@
 - no lines are manually skipped in coverage calculations
 
 Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
 
-Current coverage statistics can be viewed here:
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros2_rcutils_src/)
 
-TODO FIXME
+### Performance [4.iv]
 
-### Performance
+`rcutils` follows the recommendations for performance testing of C code in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#performance), and opts to do performance analysis on each release rather than each change.
+Soon to be defined if this package requires performance testing and how addresses this topic.
 
-TODO fix link
+### Linters and Static Analysis [4.v]
 
-`rcutils` follows the recommendations for performance testing of C code in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#performance-c), and opts to do performance analysis on each release rather than each change.
+`rcutils` uses and passes all the ROS2 standard linters and static analysis tools for a C package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters-and-static-analysis). Passing implies there are no linter/static errors when testing against CI of supported platforms.
 
-TODO how to run perf tests, where do they live, etc.
+## Dependencies [5]
 
-TODO exclusions of parts of the code from perf testing listed here? or linked to?
-
-### Linters and Static Analysis
-
-TODO fix link
-
-`rcutils` uses and passes all the standard linters and static analysis tools for a C package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters-and-static-analysis).
-
-TODO any qualifications on what "passing" means for certain linters
+### Direct Runtime ROS Dependencies [5.i]
+`rcutils` has no run-time or build-time dependencies that need to be considered for this declaration.
 
-## Dependencies
+It has several "buildtool" dependencies, which do not affect the resulting quality of the package, because they do not contribute to the public library API.
 
+### Optional Direct Runtime ROS Dependencies [5.ii]
 `rcutils` has no run-time or build-time dependencies that need to be considered for this declaration.
 
-It has several "buildtool" dependencies, which do not affect the resulting quality of the package, because they do not contribute to the public library API.
-It also has several test dependencies, which do not affect the resulting quality of the package, because they are only used to build and run the test code.
+### Direct Runtime non-ROS Dependency [5.iii]
+`rcutils` has no run-time or build-time dependencies that need to be considered for this declaration.
 
-## Platform Support
+## Platform Support [6]
```

Comment:
Links to build status in each platform?

Reply:
Added in [efce310](https://github.com/ros2/rcutils/pull/240/commits/efce310a2f85e5b473acb73c29633821fa7d6bd2)